unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.jslib

```diff
@@ -1,235 +1,236 @@
+                    if (startTime !== -1) {
+                        videoData.video.currentTime = startTime
+                    }
+                })
+                    .catch(function (error) {
+                        // Playback cancelled before the video finished loading (e.g. when teleporting)
+                        // we mustn't report this error as it's harmless and affects our metrics
+                    });
+            }
+        } catch (err) {
+            // Exception!
+        }
+    },
+
+    WebVideoPlayerPause: function (videoId) {
+        const videoData = videos[Pointer_stringify(videoId)];
+        if (videoData.hlsInstance !== undefined) {
+            videoData.hlsInstance.detachMedia(videoData.video);
         }
-    } catch (err) {
-      // Exception!
-    }
-  },
-
-  WebVideoPlayerPause: function (videoId) {
-    const videoData = videos[Pointer_stringify(videoId)];
-    if (videoData.hlsInstance !== undefined) {
-      videoData.hlsInstance.detachMedia(videoData.video);
-    }
-    videoData.video.pause();
-  },
-
-  WebVideoPlayerVolume: function (videoId, volume) {
-    videos[Pointer_stringify(videoId)].video.volume = volume;
-  },
-
-  WebVideoPlayerGetHeight: function (videoId) {
-    return videos[Pointer_stringify(videoId)].video.videoHeight;
-  },
-
-  WebVideoPlayerGetWidth: function (videoId) {
-    return videos[Pointer_stringify(videoId)].video.videoWidth;
-  },
-
-  WebVideoPlayerGetTime: function (videoId) {
-    return videos[Pointer_stringify(videoId)].video.currentTime;
-  },
-
-  WebVideoPlayerGetDuration: function (videoId) {
-    return videos[Pointer_stringify(videoId)].video.duration;
-  },
-
-  WebVideoPlayerGetState: function (videoId) {
-    const id = Pointer_stringify(videoId);
-    if (!videos.hasOwnProperty(id)) {
-      return 1; //videoState.ERROR;
-    }
-    return videos[id].state;
-  },
-
-  WebVideoPlayerGetError: function (videoId) {
-    const id = Pointer_stringify(videoId);
-    var errorStr = "";
-    if (!videos.hasOwnProperty(id)) {
-      errorStr = "Video id " + id + " does not exist";
-    } else {
-      errorStr = videos[id].error;
-    }
-
-    var bufferSize = lengthBytesUTF8(errorStr) + 1;
-    var buffer = _malloc(bufferSize);
-    stringToUTF8(errorStr, buffer, bufferSize);
-    return buffer;
-  },
-
-  WebVideoPlayerSetTime: function (videoId, second) {
-    const videoData = videos[Pointer_stringify(videoId)];
-    const vid = videoData.video;
-
-    if (second === 0) {
-      vid.currentTime = 0;
-    } else if (vid.seekable && vid.seekable.length > 0) {
-      vid.currentTime = second;
-    }
-  },
-
-  WebVideoPlayerSetLoop: function (videoId, value) {
-    videos[Pointer_stringify(videoId)].video.loop = value;
-  },
-
-  WebVideoPlayerSetPlaybackRate: function (videoId, value) {
-    videos[Pointer_stringify(videoId)].video.playbackRate = value;
-  },
+        videoData.video.pause();
+        videoData.state = 7; //PAUSED
```

Comment:
can we use `videoData.state = videoState.PAUSED` here?

Reply:
videoState its not defined on that scope for some reason